app/src/pages/movies/filter/components/FilterOption.jsx

```diff
@@ -6,23 +6,25 @@ const Option = styled.div`
     padding: 10px 0;
     margin: 0 10px -3px;
     cursor: pointer;
-    border-bottom: ${props => props.selected ? '3px solid ' + VARIABLES.red : 'none'};
+    border-bottom: ${props => props.isSelected ? '3px solid ' + VARIABLES.red : 'none'};
     &:nth-child(1) {
         margin-left: 0;
     }
 `
 
 function FilterOption(props) {
     return (
-        <Option selected={props.selectedOption === props.option} onClick={props.changeSelected}>
-            {props.option}
+        <Option isSelected={props.isSelected} onClick={() => props.changeSelected(props.option)}>

```

Comment:
() => props.changeSelected(props.option) should be wrapped into useCallback here in the component

Reply:
Will be done in next PR! (next homework is related to hooks :))